### PR TITLE
Finish Streets of King's Landing pack

### PR DESCRIPTION
**Pull request by DukeTax**

### CODE REVIEW

File: packs/SoKL.json [hunk 1]
```diff
@@ -53,7 +158,125 @@
...
+            "type": "location",
+            "name": "Summer Sea Port",
+            "unique": false,
+            "faction": "martell",
+            "loyal": true,
+            "cost": 2,
+            "traits": [
+                "Dorne"
+            ],
+            "text": "+1 Income.+1 Reserve.",
```
Comment (ystros) [inline packs/SoKL.json]: Space between the modifiers

File: packs/SoKL.json [hunk 2]
```diff
@@ -2,8 +2,63 @@
...
+            "name": "White Harbor",
+            "unique": true,
+            "faction": "stark",
+            "loyal": true,
+            "cost": 3,
+            "traits": [
+                "House Manderly",
+                "The North"
+            ],
+            "text": "<b>Reaction:</b> After you win a challenge, reveal the top 2 cards of your deck. The losing opponent adds 1 of those cards to your hand. Place the other on the bottom of your deck.",
```
Comment (ystros) [inline packs/SoKL.json]: Text does not match actual card text.

File: packs/SoKL.json [hunk 3]
```diff
@@ -53,7 +158,125 @@
             "traits": [
                 "Ally"
             ],
-            "text": "Shadow (2).\n<b>Reaction:</b> After Catspaw comes out of shadows, pay X gold to choose another character with printed cost X or lower and return it to its owner's hand.",
+            "text": "Shadow (2).\n<b>Reaction:</b> After Catspaw comes out of shadows, pay X gold to choose a character with printed cost X or lower and return it to its owner's hand.",
```
Comment (ystros) [inline packs/SoKL.json]: "choose a character" should be "choose another character"

File: packs/SoKL.json [hunk 4]
```diff
@@ -53,7 +158,125 @@
...
+            "loyal": false,
+            "cost": 2,
+            "icons": {
+                "military": true,
+                "intrigue": true,
+                "power": false
+            },
+            "strength": 1,
+            "traits": [
+                "Assassin",
```
Comment (ystros) [inline packs/SoKL.json]: Does not actually have this trait.

File: packs/SoKL.json [hunk 5]
```diff
@@ -89,6 +325,24 @@
...
+            "faction": "neutral",
+            "plotStats": {
+                "income": 4,
+                "initiative": 5,
+                "claim": 1,
+                "reserve": 5
+            },
+            "traits": [
+                "Edict",
+                "Summer"
```
Comment (ystros) [inline packs/SoKL.json]: Does not actually have the Summer trait.

**Comment by DukeTax** [comment]
Thanks!

**Pull request merged**